.travis.yml

```diff
@@ -9,6 +9,8 @@ bundler_args: --without development
 cache: bundler
 
 before_install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install 4
+  - npm install -g yarn
```

Comment:
My try would be to remove those two lines, since I don't think they are needed. It's very surprising to me that the build fails without them.